### ISSUE TITLE
Log the version of Pester that is used to run tests at the top of the output

### ISSUE THREAD

**Issue by KirkMunro** [pester/Pester#1342]
## General summary of the issue

When Pester tests pass on one machine and fail on another, users must be able to easily identify the version of Pester used in both cases.

## Detailed request

Since Pester is software like any other software, it can contain bugs. For this reason, all Pester output should log the version of Pester used to generate that output at the top of the output. That way, when there are discrepancies (the tests work on my machine), the Pester version can be checked to make sure that something that appears off isn't due to a bug that may have been fixed already in Pester.

## Possible Solution

Please consider adding the version of Pester used to run tests to the top of the output for the reasons outlined above.

## Context

I'm facing a scenario where a test that runs successfully locally fails in ADO CI builds, and I don't know why. I'd like to compare the version of Pester they use with the one I use and make sure I'm using the same version to test locally so that I can verify whether or not the issue has to do with Pester.

**Comment by nohwnd** [comment]
```
    ____            __           
   / __ \___  _____/ /____  _____
  / /_/ / _ \/ ___/ __/ _ \/ ___/
 / ____/  __(__  ) /_/  __/ /    
/_/    \___/____/\__/\___/_/                                  
Pester v4.8.0
```

So it is finally time to have some ascii art banner? 🙌

**Comment by nohwnd** [comment]
I think it should be enough to edit the [start message](https://github.com/pester/Pester/blob/ead3cbd67e31b2994192b060558395e2313af859/Functions/Output.ps1#L3) to include the version of the module (the ascii art is unnecessary imho). Will you send a PR?

**Comment by KirkMunro** [comment]
Sure, I'll send a PR for that. Just give me a bit to finish what's in front of me now.

**Comment by KirkMunro** [comment]
And I like the ASCII art. Unnecessary, sure, but retro is cool.

**Comment by nohwnd** [comment]
Should be implemented in the v5 as well. Not sure if simple forward merge will do the trick. So I am keeping this open even though it is implemented for v4.

**Comment by martin9700** [comment]
Think it would make sense to also include it in the Passthru object.

**Comment by fflaten** [comment]
Still sure about the ASCII-art in v5? 😅🤘

**Comment by nohwnd** [comment]
Imho we should output the version in the detailed mode, which we don't do now. The ascii art had surprisingly negative feedback, so let's skip that.

**Comment by nohwnd** [comment]
To implement this, just go to output.ps1 and in the write-host plugin output the version to the screen. There are few example of determining the version, e.g. when we setup the Run object. It should print the pre-release part as well, and print only in Detailed and Diagnostic view.

**Comment by ArmaanMcleod** [comment]
Hi @nohwnd, added a PR for this. I've added the pester version in detailed and diagnostic view.

**Comment by fflaten** [comment]
Closing, fixed in #1962. Will be released in next version.
